app/controllers/api/backyards_controller.rb

```diff
@@ -44,18 +49,19 @@ def article_params
   def build_backyard_article
     backyard_article = current_user.articles.build(article_params)
     backyard_article['backyard'] = true
+    backyard_article['status'] = 'published'
     backyard_article.save
     backyard_article
```

Comment:
Could you do `.published!` here then?

Reply:
no)) we did it first time, but Active Record dint like it and gave an error, unfortunately I dont remember which one.

Reply:
Weird :P